scripts/pull-specs-local.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script fetches the latest public API specs for a given HCP service ($1) 
+# from the central spec repo, cloud-api.
+
+SCRIPTS_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+service=$1
+
+# Copy the latest service specs into a temporary directory in preparation for SDK generation.
+rsync -a "$HOME"/go/src/github.com/hashicorp/cloud-api/specs/"$service" "$SCRIPTS_DIR"/../temp
```

Comment:
Maybe `$GOPATH` instead of `$HOME...`?

Also nit (since I see you're installing rsync with `sudo apt-get install -y rsync`): I think `cp -r` would work just as well here?

Reply:
👍 `rsync` seems to only be needed when dealing with hcloud cloned files.